docs/resources/sys_info.md.erb

```diff
@@ -49,4 +49,16 @@ For a full list of available matchers, please visit our [matchers page](https://
 
 The `hostname` matcher tests the host for which standard output is returned:
 
-    its('hostname') { should eq 'value' }
+its('hostname') { should eq 'value' }
```

Comment:
These code samples should remain indented (or gated w/ fenceposts?) in order to have them formatted properly.

Reply:
@zenspider just added 304ce687d212cc1c23a4fb2d7f3e457b20075fe4 which fixes this